### PR TITLE
doc: add iofs release1 install and setup

### PR DESCRIPTION
**Pull request by anandaravuri**
- command to enable virtual fpga functions
 - bind pci-vfio to virtual functions
 - command to run Host Exerciser Loopback tool

### CODE REVIEW

File: doc/src/fpga_api/quick_start/readme.md [hunk 1]
```diff
@@ -393,6 +393,79 @@ Done
...
+
+Run command: lspci | grep acc
+
+```3b:00.0 Processing accelerators: Intel Corporation Device af00 (rev 01)```
+
+Number of virtual functions supported by bitstream
+
+``` 
+   cat /sys/bus/pci/devices/0000\:3b\:00.0/sriov_numvfs 
+   output: 3
```
Comment (tswhison) [inline doc/src/fpga_api/quick_start/readme.md:408]: Will it show 3 before you have echoed 3 to it?

File: doc/src/fpga_api/quick_start/readme.md [hunk 2]
```diff
@@ -393,6 +393,79 @@ Done
...
+Program IOFS Release1 bitstream on FPGA D5005 or N6010 cards and reboot system.
+
+Run command: lspci | grep acc
+
+```3b:00.0 Processing accelerators: Intel Corporation Device af00 (rev 01)```
+
+Number of virtual functions supported by bitstream
+
+``` 
+   cat /sys/bus/pci/devices/0000\:3b\:00.0/sriov_numvfs
```
Comment (tswhison) [inline doc/src/fpga_api/quick_start/readme.md:407]: We can remove the backslash ( \ ) characters before the colons: should be 0000:3b:00.0

File: doc/src/fpga_api/quick_start/readme.md [hunk 3]
```diff
@@ -393,6 +393,79 @@ Done
...
+   output: 3
+```
+
+Enable FPGA virtual functions
+
+``` 
+   sudo sh -c "echo 3 > /sys/bus/pci/devices/0000\:3b\:00.0/sriov_numvfs"
+```
+
+List of FPGA PF and VF's
```
Comment (tswhison) [inline doc/src/fpga_api/quick_start/readme.md:417]: Write Physical Functions (PFs) and Virtual Functions (VFs).

File: doc/src/fpga_api/quick_start/readme.md [hunk 4]
```diff
@@ -393,6 +393,79 @@ Done
...
+
+``` 
+3b:00.0 Processing accelerators: Intel Corporation Device af00 (rev 01)
+3b:00.1 Processing accelerators: Intel Corporation Device af01 (rev 01)
+3b:00.2 Processing accelerators: Intel Corporation Device af01 (rev 01)
+3b:00.3 Processing accelerators: Intel Corporation Device af01 (rev 01)
+``` 
+
+FPGA VF1/3b:00.1/Host exerciser loopback Accelerator guid 56E203E9-864F-49A7-B94B-12284C31E02B
```
Comment (tswhison) [inline doc/src/fpga_api/quick_start/readme.md:426]: Will they always appear in this order?  If not, we should give instructions on how to determine the AFU ID from sysfs.

File: doc/src/fpga_api/quick_start/readme.md [hunk 5]
```diff
@@ -393,6 +393,79 @@ Done
...
+
+FPGA VF2/3b:00.2/Host exerciser memory Accelerator guid 8568AB4E-6bA5-4616-BB65-2A578330A8EB
+
+FPGA VF3/3b:00.3/Host exerciser hssi Accelerator guid 43425ee6-92b2-4742-b03a-bd8d4a533812
+
+
+
+
+Bind pcie-vfio dirver to FPGA virtual functions
```
Comment (tswhison) [inline doc/src/fpga_api/quick_start/readme.md:435]: s/dirver/driver/

File: doc/src/fpga_api/quick_start/readme.md [hunk 6]
```diff
@@ -393,6 +393,79 @@ Done
...
+
+``` 
+   sudo opaevfio  -i 0000:3b:00.1 -u userid -g userid
```
Comment (tswhison) [inline doc/src/fpga_api/quick_start/readme.md:438]: -g groupid

File: doc/src/fpga_api/quick_start/readme.md [hunk 7]
```diff
@@ -393,6 +393,79 @@ Done
...
+```
+
+Unbind pcie-vfio dirver to FPGA virtual functions
+
+``` 
+   sudo opaevfio  -r 0000:3b:00.1 -d None
```
Comment (tswhison) [inline doc/src/fpga_api/quick_start/readme.md:444]: We should omit the -d None

File: doc/src/fpga_api/quick_start/readme.md (hunk 1 above)
Comment (anandaravuri) [reply doc/src/fpga_api/quick_start/readme.md:408]: fixed , replaced \ with :

File: doc/src/fpga_api/quick_start/readme.md (hunk 2 above)
Comment (anandaravuri) [reply doc/src/fpga_api/quick_start/readme.md:407]: fixed , replaced \ with :

File: doc/src/fpga_api/quick_start/readme.md (hunk 3 above)
Comment (anandaravuri) [reply doc/src/fpga_api/quick_start/readme.md:417]: listed PF's and VF's supported in bitstream
fixed

File: doc/src/fpga_api/quick_start/readme.md (hunk 5 above)
Comment (anandaravuri) [reply doc/src/fpga_api/quick_start/readme.md:435]: fixed

File: doc/src/fpga_api/quick_start/readme.md (hunk 6 above)
Comment (anandaravuri) [reply doc/src/fpga_api/quick_start/readme.md:438]: fixed

File: doc/src/fpga_api/quick_start/readme.md (hunk 7 above)
Comment (anandaravuri) [reply doc/src/fpga_api/quick_start/readme.md:444]: removed -d None, fixed

File: doc/src/fpga_api/quick_start/readme.md (hunk 4 above)
Comment (anandaravuri) [reply doc/src/fpga_api/quick_start/readme.md:426]: updated doc with fpgainfo port, it lists AFU id

**Review by tswhison — APPROVED** [review]

**Pull request merged**